iqm5q.py

```diff
@@ -180,7 +181,15 @@ def create(runcard_path=RUNCARD):
 
     # create qubit objects
     runcard = load_runcard(runcard_path)
-    qubits, couplers, pairs = load_qubits(runcard, FOLDER)
+    qubits, couplers, pairs = load_qubits(runcard)
+    if with_kernels and (FOLDER / "kernels.npz").is_file():
+        kernels = Kernels.load(path=FOLDER / "kernels.npz")
+        for q in kernels.data.keys():
```

Comment:
```suggestion
        for q in kernels.data:
```
I think this is not needed.